tasks/us/census/acs.py

```diff
@@ -202,8 +202,8 @@ def targets(self):
     def columns(self):
         input_ = self.input()
         cols = OrderedDict([
-            ('geoidsl', input_['tiger'][self.geography + GEOID_SUMLEVEL_COLUMN]),
-            ('geoidsc', input_['tiger'][self.geography + GEOID_SHORELINECLIPPED_COLUMN]),
+            ('geoidsl', input_['tiger'][self.geography + '_{}'.format(self.year) + GEOID_SUMLEVEL_COLUMN]),
```

Comment:
I'm not sure about this `self.geography + '_{}'.format(self.year)`. How come is the year now explicit but it wasn't before and nothing is removed? It's ok for 2016, but does it mean that old runs of the task didn't have `2015` in the column name but new ones would have it? Maybe it's like that but it works because "it matches" (both before the change and afterwards)?